infretis/classes/formats/formatter.py

```diff
@@ -629,15 +630,13 @@ def write(self, towrite, end="\n"):
         if self.fileh is not None and not self.fileh.closed:
             try:
                 if end is not None:
-                    self.fileh.write("{}{}".format(towrite, end))
+                    self.fileh.write(f"{towrite}{end}")
                     status = True
                 else:
                     self.fileh.write(towrite)
                     status = True
-            except (OSError, IOError) as error:  # pragma: no cover
```

Comment:
Here also IOError is removed

Reply:
They are the same since Python 3.3: https://docs.python.org/3/whatsnew/3.3.html#pep-3151